protocol/communities/manager.go

```diff
@@ -2422,11 +2314,12 @@ func (m *Manager) HandleCommunityRequestToJoinResponse(signer *ecdsa.PublicKey,
 		return nil, err
 	}
 
-	if !common.IsPubKeyEqual(community.PublicKey(), signer) && !community.IsMemberAdmin(signer) {
+	isAdminSigner := community.IsMemberAdmin(signer)
+	if !common.IsPubKeyEqual(community.PublicKey(), signer) && !isAdminSigner {
```

Comment:
Should this be an `||`?

Reply:
no...
If the signer is not a community owner and not an admin, then return an error